### ISSUE TITLE
Url not matching room you are into

### ISSUE THREAD

**Issue by Fuzzyma** [gitterHQ/gitter#1717]
I found #1410 which is similar

Whenever I reload the page, the url indicates that a certain room is loaded.
However, the rest of the page (highlighted chat in sidear, chat content) is in another room.

When I KNOW click on the room which matches the url, nothing happens (I guess the script things that I am already in that room because the url matches).
However, when I click on the room which is highlighted atm, the page reloads and the url changes.

NOW I can change to the room whos url was initially displayed.

Here is a picture:
![unbenannt](https://cloud.githubusercontent.com/assets/3540542/26625377/310e7802-45f4-11e7-943c-8fdc23375773.PNG)

**Comment by MadLittleMods** [comment]
Seems like a duplicate of https://github.com/gitterHQ/gitter/issues/1254

**Comment by grahamperrin** [comment]
> … click on the room which is highlighted atm, the page reloads and the url changes. …

For me (with Firefox) that does not occur. Maybe the same underlying bug but the symptoms and workaround differ.

**Comment by Fuzzyma** [comment]
@grahamperrin You are right. I doublechecked. The page does not reload. Only the url changes.
